.projenrc.ts

```diff
@@ -64,4 +65,52 @@ project.addTask('integ:update', {
   receiveArgs: true,
 });
 
+const workflow = project.github!.addWorkflow('upgrade-aws-cdk');
+workflow.on({
+  schedule: [{ cron: '0 0 * * *' }],
+  workflowDispatch: {},
+});
+workflow.addJob('create-pr', {
+  permissions: {
+    contents: JobPermission.WRITE,
+    pullRequests: JobPermission.WRITE,
+  },
+  runsOn: ['ubuntu-latest'],
+  steps: [
+    {
+      uses: 'actions/checkout@v4',
+    },
+    {
+      name: 'Install dependencies',
+      run: 'yarn install',
```

Comment:
Thanks for this PR! Should we use npm?

see: https://github.com/open-constructs/aws-cdk-library/issues/9

Reply:
Oops, thanks!